COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
+
+Follow these instructions to work on Windows with Visual Studio:
+
+1. First, clone the `cbmc` repository locally.
```

Comment:
:pick: This isn't a Visual Studio specific instruction. Visual Studio 2019 features git integration. So it would be great to include instructions on cloning CBMC using that integration - [Visual Studio git documentation](https://docs.microsoft.com/en-us/visualstudio/version-control/git-with-visual-studio?view=vs-2019#clone-an-existing-git-repository)